python/import_util.py

```diff
@@ -0,0 +1,36 @@
+import mgp
+import json as js
+import sys
+
+
+@mgp.write_proc
+def json(ctx: mgp.ProcCtx, path: str) -> mgp.Record():
+    """
+    Procedure to import the local JSON created by the export_util.json procedure.
+
+    Parameters
+    ----------
+    path : str
+        Path to the JSON that is being imported.
+    """
+    file = open(path)
+    list = js.load(file)
+    vertex_ids = dict()
+
+    for el in list:
+        if el["type"] == "node":
+            vertex = ctx.graph.create_vertex()
+            vertex_properties = vertex.properties
+            for key, value in el["properties"].items():
```

Comment:
Check whether the key 'properties' exist in the dictionary, if not, throw an exception

Reply:
It applies to other keys here